vm/errors.go

```diff
@@ -50,6 +50,9 @@ var ErrInvalidMinStakeValue = errors.New("invalid min stake value")
 // ErrInvalidNodePrice signals that an invalid node price was provided
 var ErrInvalidNodePrice = errors.New("invalid node price")
 
+// ErrInvalidMinStepValue signals that an invalid min step value was provided
+var ErrInvalidMinStepValue = errors.New("invalid min step value")
```

Comment:
Should add comments on exported items in vm/consts.go

Reply:
that is from a pull request from Radu already merged in development./